runtime/lua/vim/_watch.lua

```diff
@@ -283,6 +283,13 @@ function M.fswatch(path, opts, callback)
     '/.git/',
     path,
   }, {
+    stderr = function(err, data)
+      if err then
+        error(err)
+      end
+
+      vim.notify(data, vim.log.levels.ERROR)
```

Comment:
```suggestion
      if data and #vim.trim(data) > 0 then
        vim.notify(data, vim.log.levels.ERROR)
      end
```
For me it prevents from displaying empty notification.